integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -499,6 +499,9 @@ object CarbonDataRDDFactory {
       LoadTablePostExecutionEvent(sqlContext.sparkSession,
         carbonTable.getCarbonTableIdentifier,
         carbonLoadModel)
+      // REVIEW_COMMENT: I think here we should know the datamap loading is failed or sucess,
+      // if it is failed, loading of main table should fail also. So that user can know it is
```

Comment:
Yes jacky if any child table data loading is failing it will fail the complete data loading process , it is handled